2269/2/rapers_list.rb

```diff
@@ -0,0 +1,22 @@
+# :reek:DuplicateMethodCall
+# :reek:UtilityFunction
+# :reek:TooManyStatements
+# rubocop:disable Metrics/AbcSize
+# rubocop:disable Metrics/MethodLength
+def get_rapers(destination)
+  hash = {}
+  Dir.entries(destination).each do |file|
```

Comment:
that's file_name, not a file object.
ps.sorry being nerdy here, but that's part of the proficiency.

Reply:
changed a method name